### PR TITLE
Adding initial documentation to readme

### LINKED ISSUE #1: Improve readme
Add some description about what composer does, how it is configured and how it works to the readme

### PR DESCRIPTION
**Pull request by thovid**
resolves #1

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,91 @@
...
+
+Routes can be configured via the configuration file using [HCON - human optimised configuration notation](https://github.com/lightbend/config/blob/master/HOCON.md). 
+
+*Example:*
+```
+{ path: "/", method: "GET", type: "PROXY", target: "https://www.rewe-digital.com/" }
+```
+Configures a *proxy route* for the path `\`and method `GET` that forwards all requests to the target `https://rewe-digital.com`.
+
+Path parameters can be captured using the syntax of the underlying routing eninge (https://github.com/danielnorberg/rut), for example
```
Comment (AndreasKl) [inline readme.md]: eninge => engine

File: readme.md [hunk 2]
```diff
@@ -1,6 +1,91 @@
...
+  </rewe-digital-content>
+</body>
+```
+In this example, an *include* tag pointing to the above content would be replaced with the markup enclosed by the *content* tag.
+
+Using content tags to mark the content to be included allows the response of some content service to contain a fully working html document (including for example a html head element), thus allowing a team to test their content page without having to actually include it into some template. 
+
+**Specify assets required for included content**
+
+Content fragments usually need some assets like css and/or javascript to work. For this. content responses can mark asset links using a special data attribute. The name of this attribute is *data-rd-options* and can be changed vie configuration. If an asset link in the html head of the content response contains the `data-rd-options` attribute with value `include`, this links is included in the head of the document returned to the caller.
```
Comment (AndreasKl) [inline readme.md]: vie => via

Comment (AndreasKl) [reply readme.md]: , this links => , this link

**Review by AndreasKl — APPROVED** [review]
Found a few typos, would be nice if you could fix them.

**Comment by thovid** [comment]
Thanks for the review. I've amended the fix and will merge the PR.

**Pull request merged**